### ISSUE TITLE
Notebooks need to be updated to install python packages and get instance name

### ISSUE THREAD

**Issue by anouri** [IBMStreams/sample.starter_notebooks#10]
notebooks need to be updated to install python packages and get instance name
After the installation of python packages it is needed to set the path of python packages to user path.
 for example
```
!pip install --user --upgrade streamsx
!pip install --user --upgrade streamsx.hdfs
 ```
After the installation set manually the order of path to user path :
```
import sys
#you cam find the user path 
for e in sys.path:
    print(e)

sys.path.insert(0, '/home/wsuser/.local/lib/python3.6/site-packages')
```

More details in:

https://github.ibm.com/PrivateCloud-analytics/Zen/issues/9877

### Update the section of  "Add credentials for the IBM Streams service"
The "Connect to instance" button is not more available in V5.2.0.0-CP4Data

This section has to be change to get the streams instance name from My instances.

In order to submit a Streams application you need to provide the name of the Streams instance.

    From the navigation menu, click My instances.
    Click the Provisioned Instances tab.
    Update the value of streams_instance_name in the cell below according to your Streams instance name.

**Comment by anouri** [comment]
HBASE notebook has been updated:
- The notebook uses a connection string in stead of connection type. (No HBASE connection type is more available in V5.2.0.0-CP4Data)
- The connection string can be inserted in a edit window.
- The connection string contains the hostname and the port  of HBASE server.
- The download of latest version of streamsx.hbase toolkit has been added to the notebook.

**Comment by anouri** [comment]
The section of upgrading of Python packages for HDFS and HBASE notebooks moved to the top of the notebook.
We have to run in the first step the following commands :
```
#!pip install --user --upgrade streamsx
#!pip install --user --upgrade streamsx.TOOLKIT_NAME

```
And then we have to set manually the Python path for user.
```
import sys
sys.path.insert(0, '/home/wsuser/.local/lib/python3.6/site-packages')
```
After these steps we can set up the streams instance.
